test/metric/app_signals_util.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package metric
+
+import (
+	"github.com/aws/aws-sdk-go-v2/aws"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+)
+
+var (
```

Comment:
These variables seem specific to the `app_signals_test.go`. Why are they here? Are they used elsewhere?

Reply:
Added to utils as a common place to reuse.